Rat pAI and wide pAI sprite support

TalkingCactus opened the pull request:
- adds a large rat as a pAI chassis

- adds support for 64x64 pAI sprites

TalkingCactus committed:
rat pai

TheFurryFeline commented on code/modules/mob/living/silicon/pai/pai.dm:
```diff
@@ -34,7 +34,13 @@
 		"Rabbit" = "rabbit",
 		"Bear" = "bear",  //VOREStation Edit
-		"Raccoon" = "raccoon" //Chompstation Edit
+		"Raccoon" = "raccoon", //Chompstation Edit
```
Add a comma to the end of edit comments inside of a list. Travis usually screeches if there aren't any. Actually, since this is a main file, and we have a _vr file already, maybe move it there.

TalkingCactus replied:
a global list can't be re-initialized in another file, unfortunately. I'd have to rig up a complicated system to add our custom content to these lists during the pregame initialization

`code\modules\mob\living\silicon\pai\pai_vr.dm:8:error: possible_chassis: re-initialization of global var`

`code\modules\mob\living\silicon\pai\pai_vr.dm:22:error: possible_say_verbs: re-initialization of global var`

TheFurryFeline commented on code/modules/mob/living/silicon/pai/pai_vr.dm:
```diff
@@ -30,7 +30,13 @@
 	else if(people_eaten && resting)
 		icon_state = "[chassis]_rest_full"
-
+	//CHOMPStation edit
```
No need to have a //Chompstation edit for _vr files.

TheFurryFeline commented on code/modules/mob/living/silicon/pai/pai_vr.dm:
```diff
@@ -42,3 +48,10 @@
 		icon_state = "[chassis]_full"
 	else if(people_eaten && resting)
 		icon_state = "[chassis]_rest_full"
+	//CHOMPStation edit | catch instances where the icon never updated for whatever reason
```
Ditto here about Chompstation edit comments.

TheFurryFeline reviewed (changes requested):
Couple of things you might wanna do here.

TalkingCactus committed:
review edits and bug fix

removes the "are you sure" dialog, which could be abused to keep an invisible sprite in some cases. was honestly just bloat code anyways

TalkingCactus commented:
all set, now

TheFurryFeline commented on code/modules/mob/living/silicon/pai/pai.dm:
```diff
@@ -34,7 +34,8 @@
 		"Rabbit" = "rabbit",
 		"Bear" = "bear",  //VOREStation Edit
-		"Raccoon" = "raccoon" //Chompstation Edit
+		"Raccoon" = "raccoon", //Chompstation Edit
```
Still might want to add a comma at the end of 'Edit' in the comments here, since it's still well within a list.

TheFurryFeline reviewed (approved):
Looks good, travis is approving here: https://travis-ci.org/CHOMPstation/CHOMPstation/builds/567641718
